### ISSUE TITLE
Installation with Homebrew

### ISSUE THREAD

**Issue by FloLecoeuche** [reorproject/reor#331]
**Is your feature request related to a problem? Please describe.**
Hello,
Usually i install and update my apps with Homebrew (on macOS).

**Describe the solution you'd like**
I wish i could install Reor with Homebrew.

**Comment by samlhuillier** [comment]
Yes I'd love to have it on homebrew - could you help with this @FloLecoeuche?
